### ISSUE TITLE
AnimationGroups don't work with latest release 0.64.0-alpha.48 and @babylonjs/core-rc.1

### ISSUE THREAD

**Issue by f4z3k4s** [BabylonJS/BabylonReactNative#356]
**For questions, please use the [forum](https://forum.babylonjs.com)**.

**Describe the bug**

```ts
// animation is of type AnimationGroup
animation.start(false, 1, 0, 1);
``` 

doesn't run animations as it used to with:
0.64.0-alpha.47 and @babylonjs/core-alpha.65

I've just found out that it didn't work with @babylonjs/core-beta.11 either (connecting to the issue I've opened yesterday #354)

**To Reproduce**
1. Use latest version of this package
2. Try to run AnimationGroups with the `start` method

**Expected behavior**
Animation group runs just as before.

**Screenshots**

**Other**
 - Platform: Android

**Comment by bghgary** [comment]
This is a Babylon.js bug. I can repro in the playground.

Here is the repro: https://playground.babylonjs.com/#NIH5E6

Works in 4.2.1 but not in 5.0.

**Comment by bghgary** [comment]
Wait, I take that back. This is not a bug. It's a change on the Babylon.js that makes the loading of glTF animations be 60fps instead of 1fps. The equivalent thing is to do [this](https://playground.babylonjs.com/#NIH5E6#1):

```
result.animationGroups[0].start(false, 1, 0, 60);
```

You can also set the `targetFps` on the loader back to 1 if you want the old behavior.
https://playground.babylonjs.com/#NIH5E6#2

Please use the forum next time before filing an issue. Thanks!

**Comment by f4z3k4s** [comment]
Sorry, you're right. I should've used the forum and I should've checked the babylonjs changelog. What was strange to me is that I didn't increase major version, I just upgraded from alpha to rc, and it used to work with alpha. I also upgraded @babylonjs/react-native version at the same time. I was ignorant, alpha is alpha because it can break things, just as it did with [this commit](https://github.com/BabylonJS/Babylon.js/commit/fa4157db3169f4504f572aaa1a9b1fe8dfd632fd) as you pointed out. I'll pay more attention next time, thanks for your suggestion, it works. :) Cheers!
